src/applications/vaos/components/calendar/CalendarWeekdayHeader.jsx

```diff
@@ -17,7 +35,13 @@ const CalendarWeekdayHeader = () => (
           role="columnheader"
           className="vaos-calendar__weekday vads-u-font-weight--bold vads-u-text-align--center vads-u-margin-bottom--0p5"
         >
-          {day}
+          <span className="vads-u-display--none small-screen:vads-u-display--inline">
+            {day.name}
+          </span>
+          <span className="small-screen:vads-u-display--none">
+            <span aria-hidden="true">{day.abbr}</span>
+            <span className="sr-only">{day.name}</span>
```

Comment:
VoiceOver reads out the day of the week truncation correctly for all the days but Monday. This gets it to say the right name.

It seems to say the name of the day twice no matter what content is in here, so I'm not sure what's up with that.